release/release_tests.yaml

```diff
@@ -5497,6 +5503,35 @@
         cluster_env: stress_tests/stress_tests_app_config.yaml
         cluster_compute: stress_tests/smoke_test_compute_gce.yaml
 
+
+- name: stress_test_many_runtime_envs
+  group: core-daily-test
+  working_dir: nightly_tests
+
+  python: "3.7"
+  frequency: nightly
+  team: core
+  cluster:
+    cluster_env: stress_tests/stress_tests_app_config.yaml
```

Comment:
please use byod version of the test, see the test above this

Reply:
done.